File: api/src/main/java/com/edukaiquedev/api/dto/VotoResponse.java
package com.edukaiquedev.api.dto;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class VotoResponse {
    private String mensagem;
    private boolean sucesso;
    // null = sucesso; 1 = token inválido; 2 = já votou; 3 = usuário não encontrado
    private Integer codigoErro;
}

File: android-app/app/src/main/java/com/edukaiquedev/android_app/api/VotoResponse.kt
package com.edukaiquedev.android_app.api

import com.google.gson.annotations.SerializedName

data class VotoResponse(
    @SerializedName("mensagem")   val mensagem: String,
    @SerializedName("sucesso")    val sucesso: Boolean,
    @SerializedName("codigoErro") val codigoErro: Int?
)
